packages/dashmate/templates/platform/dapi/envoy/envoy.yaml.dot

```diff
@@ -168,6 +168,7 @@ static_resources:
               "@type": type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.DownstreamTlsContext
               common_tls_context:
                 tls_certificates:
+                  alpn_protocols: [ "h2,http/1.1" ]
```

Comment:
Do we need it only for self-signed? Look at the tempting.

Reply:
It works for signed as well, Mobile needs it not DAPI.

Reply:
I've already deployed this onto hp-masternode-29 and got Hash to check, it was working for him with the zerossl cert

Reply:
No I mean there is another branch of if else condition where this setting is missed

Reply:
For some reason the gRPC Java library requires that `h2` be made available via ALPN when using TLS/SSL communication with servers that have certificates.